scripts/xtensa-build-zephyr.py

```diff
@@ -588,6 +588,12 @@ def build_platforms():
 			tools_output_dir,
 			symlinks=True, ignore_dangling_symlinks=True, dirs_exist_ok=True)
 
+	# mtrace
+	shutil.copytree(pathlib.Path(SOF_TOP) /
+			  "tools" / "mtrace",
```

Comment:
Please append this file to `src_dest_list` instead, as recommended by the comment line 577. We already have two different code sections to install files, we don't need a third one.

Reply:
Updated.